src/WebJobs.Script.WebHost/Diagnostics/MetricsEventManager.cs

```diff
@@ -231,13 +232,19 @@ public void Flush()
 
         private void FlushMetrics()
         {
-            if (QueuedEvents.Count == 0)
+            if (QueuedEvents.IsEmpty)
```

Comment:
Count takes a lock, whereas in https://github.com/drewnoakes/corefx/commit/67e2891b64edd510e4245dc86434f50859776472 a fastpath check was added. So this looks like a good change :)